spec/policies/vacancy_policy_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe VacancyPolicy, type: :policy do
```

Comment:
@sk8higher good but I prefer use dsl for policy 
https://actionpolicy.evilmartians.io/#/testing